enterprise/models.py

```diff
@@ -652,14 +652,15 @@ def clear_pending_registration(self, email, *course_ids):
         else:
             PendingEnrollment.objects.filter(user=pending_ecu, course_id__in=course_ids).delete()
 
-    def notify_enrolled_learners(self, catalog_api_user, course_id, users):
+    def notify_enrolled_learners(self, catalog_api_user, course_id, users, bulk_enrollment=False):
```

Comment:
Suggestion: I would change the names in several places in this PR. I know we call the Quokkas feature 'bulk enrollment', but the reason we have this new template isn't "single vs bulk",  it's "Self Enrolled" or "Admin Enrolled". Reading the code for all of this stuff, it would be a lot clearer that the important part to this feature is that the templates change based on who did the enrolling. Thoughts?

Reply:
yeah this is a great point, self vs admin is the key distinction indeed. going to do this...

Reply:
done!